packages/common/src/requests/update_user_request.ts

```diff
@@ -1,5 +1,8 @@
-import { InferType } from "yup";
-import { updateUserSchema } from "@cryptify/common/src/validations/update_user_schema";
-
-// eslint-disable-next-line @typescript-eslint/no-empty-interface
-export interface UpdateUserRequest extends InferType<typeof updateUserSchema> {}
+export type UpdateUserRequest = {
+    userId: number;
+    firstName?: string;
+    lastName?: string;
+    email?: string;
+    password?: string;
+    areNotificationsEnabled?: boolean;
+};
```

Comment:
Can you make this an interface still so that the auto class generator will pick up on it.
```suggestion
export interface UpdateUserRequest {
    userId: number;
    firstName?: string;
    lastName?: string;
    email?: string;
    password?: string;
    areNotificationsEnabled?: boolean;
}
```